### PR TITLE
Various small doc updates

### PR DESCRIPTION
**Pull request by kimausloos**

### CODE REVIEW

File: docs/configuration-management.md [hunk 1]
```diff
@@ -21,13 +21,13 @@ You can set multiple environment variables at once:
...
 
-When setting variables with whitespace, you need to escape the whitespace:
+When setting variables with whitespaces, you need to escape them:
 
 ```shell
 dokku config:set node-js-app KEY=\"VAL\ WITH\ SPACES\"
 ```
 
-When setting or unsetting environment variables, you may wish sometimes to avoid an application restart. This is useful when developing plugins or when setting multiple environment variables in a scripted manner. To do so, use the `--no-restart` flag:
+When setting or unsetting environment variables, you sometimes may wish to avoid an application restart. This is useful when developing plugins or when setting multiple environment variables in a scripted manner. To do so, use the `--no-restart` flag:
```
Comment (josegonzalez) [inline docs/configuration-management.md]: Just remove `sometimes` here.

File: docs/deployment/one-off-processes.md [hunk 2]
```diff
@@ -4,8 +4,9 @@ Sometimes you need to either inspect running containers or run a one-off command
 
 ## Run a command in an app environment
 
-```
-run <app> <cmd>                                  Run a command in the environment of an application
+```shell
+# Run a command in the environment of an application
+run <app> <cmd>
```
Comment (josegonzalez) [inline docs/deployment/one-off-processes.md]: Do not reformat these codeblocks please. Will mean updating the help message requires more work.

**Comment by josegonzalez** [comment]
Protip: Avoid test runs using `[ci skip]` in your commit messages :)

**Comment by callahad** [comment]
I've updated the patch to address the comments above. Suggest closing in favor of #1657.

**Pull request closed without merging**